src/main/java/graphql/GraphQL.java

```diff
@@ -35,8 +38,13 @@ public GraphQL(GraphQLSchema graphQLSchema) {
 
 
     public GraphQL(GraphQLSchema graphQLSchema, ExecutionStrategy executionStrategy) {
+        this(graphQLSchema, executionStrategy, NoOpInstrumentation.INSTANCE);
+    }
+
+    public GraphQL(GraphQLSchema graphQLSchema, ExecutionStrategy executionStrategy, Instrumentation instrumentation) {
         this.graphQLSchema = graphQLSchema;
         this.executionStrategy = executionStrategy;
+        this.instrumentation = instrumentation;
     }
```

Comment:
Yet more constructors is an anti pattern I know.  I think GraphQL needs a builder

      GraphQL graphQL = GraphQL.builder(schema).
          .strategy(x)
          .instrumentation(y)
          .build()

That way the API can be evolved more easily without breaking existing client or adding yet more overloaded methods / constructors